aioxmpp/avatar/__init__.py

```diff
@@ -0,0 +1,57 @@
+########################################################################
+# File name: __init__.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+"""
+:mod:`~aioxmpp.avatar` --- User avatar support (:xep:`0084`)
+############################################################
+
+
+This module provides support for publishing and retrieving user
+avatars as per :xep:`User Avatar <84>`.
+
+Services
+========
+
+The following services are provided by this subpackage:
+
+.. autosummary::
+   AvatarServer
+   AvatarClient
+
+The detailed documentation of the classes follows:
+
+.. autoclass:: AvatarServer
+
+.. autoclass:: AvatarClient
+
+Data Representation
+===================
+
+The following class is used to describe the possible locations of an
+avatar image:
+
+.. autoclass:: AvatarSet
+
+.. autoclass:: aioxmpp.avatar.service.AbstractAvatarDescriptor
```

Comment:
Use something like:
```
.. module:: aioxmpp.avatar.service
.. currentmodule:: aioxmpp.avatar.service
.. autoclass:: AbstractAvatarDescriptor
```

instead. This can be found in other service docs too, grep for 'module::'